### ISSUE TITLE
Control Windows Update Service over GPO ADMX 

### ISSUE THREAD

**Issue by zurstegen** [EUCweb/BIS-F#314]
**Is your feature request related to a problem? Please describe.**
With the default BISF Powershell files the Windows Update service will be disabled in the preperation task. This is a problem, if in the provisioned Citrix workers the Windows Defender service should be use to update the Anti Virus Signature Update files.

**Describe the solution you'd like**
As an solution the administrator should be select over the BISF GPO ADMX file, if the Windows Update service shoud be enabled or disabled.

**Describe alternatives you've considered**
Manually comment out the PS Custom Object (Description: "Set Windows Update Service to Disabled") in the 97_PrepBISF_PRE_BaseIMage.ps1 file.

**Additional context**

**Comment by xenadmin** [comment]
Hi @zurstegen !
I just wanted to let you know, that I think that your request is misleading. Windows Defender does **not** need the Windows Update service, to function properly and receive security intelligence updates. We are currently transforming several customers from traditional AV solutions, to Windows Defender, and I thought the same, but I quickly discovered, that the recommended solution from Microsoft doesn't rely on the Windows Update service, but **on a SMB share**. Please read the following documentations, and you will see, that the disabled service is no problem whatsoever. 
https://docs.microsoft.com/en-us/microsoft-365/security/defender-endpoint/configure-endpoints-vdi?view=o365-worldwide
https://www.mycugc.org/blogs/nishith-gupta1/2020/08/31/microsoft-defender-for-citrix-virtual-apps-and-des
https://techcommunity.microsoft.com/t5/microsoft-defender-for-endpoint/configuring-microsoft-defender-antivirus-for-non-persistent-vdi/ba-p/1489633
https://docs.microsoft.com/en-us/microsoft-365/security/defender-endpoint/deployment-vdi-microsoft-defender-antivirus?view=o365-worldwide
To sum up, I strongly believe, that it's still a best practice, to disable Windows Update service in a MCS/PVS clone, and there is no need to change BIS-F. And if I remember correctly, Citrix Optimizer also disables Windows Update service.

**Comment by zurstegen** [comment]
Hi @xenadmin!

Thanks for the detailed introduction. The way to point the defender service to a SMB share sounds very interest and we will test it. 

I agree with you that on MCS/ PVS the Windows Update service normally should be disabled, but in my scenario we didn't found a solution to get around this. The mycugc article could be solution for us!

Thanks!
